### ISSUE TITLE
Training not proceeding

### ISSUE THREAD

**Issue by liaoluke** [AlexeyAB/darknet#1320]
Hi, I've used the YoloMark program and labeled some images for training, and am trying to train my custom model. However, when I run the training the loss graph shows up briefly and then disappears, and the program terminates seemingly without doing any training. It does save the weights to backup/ and doesn't give an error though.

Console output shows:
Loading weights from yolov3_pretrain.weights...
 seen 32
Done!
Learning Rate: 0.0005, Momentum: 0.9, Decay: 0.0005
 If error occurs - run training with flag: -dont_show
Saving weights to backup//yolov3_train_final.weights

Do you know what's going wrong here?

**Comment by liaoluke** [comment]
I think I found the issue, it was that the weights file was pretrained and had a current batch number greater than the maximum batch number set in the cfg file. I changed that and now it trains.

**Comment by git-sohib** [comment]
Hi @liaoluke I am facing the same issue here. my pretrained weight batch number is 180000 and max_batches in cfg file is 193000. What do you think the problem is then? I tried to increase max_batches with no luck.
